changes/serializers.py

```diff
@@ -127,3 +127,17 @@ class Data(serializers.Serializer):
         address = PhoneNumberField(country_code="ZA")
 
     data = Data()
+
+
+class SeedMessageSenderFailedMsisdnHookSerializer(serializers.Serializer):
+    class Hook(serializers.Serializer):
+        id = serializers.IntegerField()
+        event = serializers.ChoiceField(choices=["identity.no_address"])
+        target = serializers.CharField()
+
+    hook = Hook()
+
+    class Data(serializers.Serializer):
+        registrant_id = serializers.UUIDField()
```

Comment:
In the webhook: https://github.com/praekeltfoundation/seed-message-sender/blob/develop/message_sender/tasks.py#L218-L223 , this field is named `to_identity`, not `registrant_id`.